### ISSUE TITLE
Subdl search over the internet feature  allow sign in 

### ISSUE THREAD

**Issue by erikk333** [recloudstream/cloudstream#1097]
### Describe your suggested feature

The recent update in cloudstream allows to search for subtitles obtained from subdl. But since subdl has a rate limitation of 2000 subtitles a day per account this will result in rate limits. Allow users to sign in to fix the rate limitation issue. 
![Screenshot_2024-05-19-18-18-50-514_org telegram messenger](https://github.com/recloudstream/cloudstream/assets/147526653/2c5d299c-8284-488d-ae15-b119616da296)
![Screenshot_2024-05-19-18-20-01-617_com lagradost cloudstream3 prerelease](https://github.com/recloudstream/cloudstream/assets/147526653/e40e08f5-3ded-4fc0-a02f-e43bdd3d40fd)

### Other details

It  Doesn't  load for me and the application is updated. I'm not confident it's related to the rate limitation 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by KingLucius** [comment]
There is no way to sign in yet in their api

**Comment by erikk333** [comment]
Is it possible to let users enter their own api key? I think a stremio addon allowed that

**Comment by erikk333** [comment]
<https://stremio-subdl.vercel.app/configure>

**Comment by KingLucius** [comment]
SubDL developer is working on official login support via API requests. once he is done, I will update.
API Key from SubDL side is for on developer level not the user level, and we have to respect their wishes of implementation.

**Comment by KingLucius** [comment]
![Screenshot_20240520_174002_com.lagradost.cloudstream3.prerelease_edit_31177209906023.jpg](https://github.com/recloudstream/cloudstream/assets/17493070/60257e15-c178-4e15-a14a-701e9d572f9e)

What happens on your end has nothing to do with rate limit

**Comment by erikk333** [comment]
It worke fine
